### DISCUSSION TITLE
Can't access Student Developer Pack

### DISCUSSION THREAD

**Discussion by ericl3ung** [community/community#38428] [GitHub Education]
I am unable to access the developer pack. Will not let me reverify my student status.

**Comment by grahamzemel** [comment]
Hey, great question @ericl3ung! Can you be a bit more specific in the error message that's coming up when you try to verify yourself? Is it telling you that you've reached too many attempts or that you're not submitting verifiable media? 

Comment with a bit more information and I'll be sure to reach out and take another look.

**Reply by jsurya860** [reply]
![image](https://user-images.githubusercontent.com/108981982/218042483-87fe8b19-30ee-4c87-a534-d7cbc16cfa0d.png)

**Comment by ericl3ung** [comment]
My Process:

Are you a student? {I click Yes}
You must periodically re-verify your academic affiliation to continue to qualify for GitHub Education benefits. If you are having trouble accessing your benefits. {I click submit information again}
"You were last verified as a student on May 4, 2022. It is not necessary for you to re-verify at this time". {Message pop-up}

Hope that helps...

**Reply by akashrai2003** [reply]
I'm getting this thing as You were last verified as a student on Dec 30, 2022. It is not necessary for you to re-verify at this time but I'm not able to see the continue option going forward?
While I was opting for student campus benefits

**Comment by samus-aran** [comment]
Hi @ericl3ung 👋 

Are you still experiencing this? 

What benefits are you unable to access? Do you have an error message when you [visit the Benefits](https://education.github.com/benefitsl) page?

**Comment by edisonAlbaSarmiento** [comment]
Hello, I also have this problem and I cannot use the gitkraken service, I can only open public repositories but not private ones and it is the one I use the most at the university
<img width="757" alt="image" src="https://user-images.githubusercontent.com/31748119/212382762-60673834-acb0-4e9c-b085-3e155b5417d7.png">

Thanks for the help
